.gitignore

```diff
@@ -46,3 +46,4 @@ phpcs*
 *.v11.suo
 *~
 composer.phar
+spotify-test
```

Comment:
Is this necessary?

Reply:
I think so, or I wouldn't have added it.  If you think I should polish it and add it to git history, given time, that is a possibility.

Also, isn't that the purpose of ,gitignore?

Reply:
> Also, isn't that the purpose of ,gitignore?

Just curious. The .gitignore file of a project should contain just project-specific entries - mostly generated stuff like composer.lock, the vendor-directories or user-specific versions of certain files (e.g. a phpunit.xml or the ampache-config file). The goal is to keep the .gitignore as small/clean as possible.

Temporary/test files should be treated like custom editor settings/artefacts (like .idea, vim swap files, ...) and should be placed in the users global .gitignore which has to be configured in the users global .gitconfig.

Examplewise in linux:
```
/home/<username>/.gitconfig
....
[core]
    excludesfile = /home/<username>/.gitignore
....
```

Reply:
>  The .gitignore file of a project should contain just project-specific entries -

And that's what I consider spotify-test.

> ...should be placed in the users global .gitignore which has to be configured in the users global .gitconfig.

Point well taken will make the change. And you should have led the conversation with that instead of the authoritarian attitude.

Reply:
> And you should have led the conversation with that instead of the authoritarian attitude.

If I sounded authoritarian, I apologize.
I'm used to ask straight questions instead of playing the `know-it-all` and telling everyone, what to do. Many people (including me) tend to forget removing temporary entries in .gitignore. So my intention was just to point it out (in my clumsy german way).